src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -87,12 +94,24 @@ public Path saveBlobDataToTempWorkspace(InputStream mediaFile, String fileName)
 
     @Override
     public void handleKedaInvocationForMediaRequests() {
-        Optional<MediaRequestEntity> mediaRequest = mediaRequestService.retrieveMediaRequestForProcessing();
-
-        mediaRequest.ifPresentOrElse(
-            this::processAudioRequest,
-            () -> log.info("No open requests found for ATS to process.")
-        );
+        int counter = 1;
+        List<Integer> mediaRequestIdsProcessed = new ArrayList<>();
```

Comment:
in your PR description you've explained that this list list is introduced to skip picking up media requests whose files are not yet ready (in outbound store). 
can we try explaining this in the code a bit better since it's not easy to understand otherwise. Even a comment would be fine

Reply:
code comment added now